try {
    console.log(a)
} catch {
    console.log("caught an error. a is not defined")
}